.cfnlintrc.yaml

```diff
@@ -0,0 +1,2 @@
+ignore_checks:
+  - W3005
```

Comment:
Can this be a lint config for cimit stub only?

Reply:
I decided to leave it as I expect to be doing a similar thing for the orch stub and maybe others (also just moving it to the template folders  didn't work)